src/utils.ts

```diff
@@ -200,3 +200,8 @@ export function validateFormField(
       !notEnding.some((check) => value.endsWith(check)))
   );
 }
+
+// Remove all empty values from array
```

Comment:
What array? :D

Reply:
I kept this for future use, but made the comment more sensible :D